### ISSUE TITLE
[BUG] Parser doesnt support Text/Json nor application/*+json as valid types for content within request body

### ISSUE THREAD

**Issue by JohanpG** [manchenkoff/openapi3-parser#67]
**Describe the bug**
When the Open API specification contains context with text/json or application/*+json these are not recognized as valid types. 

**To Reproduce**
1. Load a api specification containing any of these types in the request body content:
```JSON "requestBody": {
                    "content": {
                        "application/json": {
                            "schema": {
                                "$ref": "#/components/schemas/..."
                            }
                        },
                        "text/json": {
                            "schema": {
                                "$ref": "#/components/schemas/..."
                            }
                        },
                        "application/*+json": {
                            "schema": {
                                "$ref": "#/components/schemas/..."
                            }
                        }
                    }
                }
```
2. Pass this file to the openapi parser
3. See error

**Expected behavior**
These types should be valid tpes and the specification file should be validated.

**System details (please complete the following information):**
 - Windows 11
 - OpenAPI / Swagger version OpenAPI 
 - Python version 3.9
 
**Additional context**
Removing these entries from the file allowed the validaation and loading of the ile into python.

**Comment by manchenkoff** [comment]
Hey @JohanpG, thanks for opening this issue! I defined only the most popular types in an enumeration, but according to [RFC6838](https://datatracker.ietf.org/doc/html/rfc6838) it looks like you can use almost everything there 😄 I probably will reconsider this approach in the next major versions. For now as a temporary solution I extended enumeration with problematic values in #68

**Comment by manchenkoff** [comment]
Published as v1.1.13

**Comment by JohanpG** [comment]
Thanks! Yeah I noticed the most common ones were covered, so as a work around I had to manually change the file to remove those types from it, but now i dont need to do that 👍
